features/ui-common/src/main/kotlin/com/wire/android/ui/common/button/WirePrimaryIconButton.kt

```diff
@@ -79,15 +78,15 @@ fun WirePrimaryIconButton(
 @Preview
 @Composable
 fun PreviewWirePrimaryIconButton() {
-    WirePrimaryIconButton({}, false, R.drawable.ic_add, 0)
+    WirePrimaryIconButton({}, false, com.google.android.material.R.drawable.m3_password_eye, 0)
```

Comment:
I suppose the new icon is to be as "default" until we have the project drawables into `ui-common` module as well, right?

Reply:
Yup, forgot to mention, this `drawable` was used for previews only in these components, that's why I replaced it on the components.

The real resource we passed from specific modules, in our case `App` or new feature modules :D